app.json

```diff
@@ -2,7 +2,7 @@
   "name": "coinfi-rails",
   "scripts": {
     "postdeploy":
-      "pg_dump $STAGING_DATABASE_URL | psql $DATABASE_URL && bundle exec rake db:migrate"
+      "pg_dump $STAGING_DATABASE_URL | psql $DATABASE_URL && bundle exec rake db:migrate; bundle exec rake redis:copy_prod"
```

Comment:
Hm, should this be `redis:make_slave` instead based on our Slack conversation?

Reply:
Ah sorry, I've been leaving comments in the asana ticket. Heroku redis doesn't allow slaveof